src/04-arrays.js

```diff
@@ -119,7 +138,10 @@ function addElementToFrontOfArray(array, element) {}
  *  removeElementFromFrontOfArray([ 10, 9, 8 ]);
  *  //> 10
  */
-function removeElementFromFrontOfArray(array) {}
+function removeElementFromFrontOfArray(array) {
+  array.shift();
```

Comment:
Letss take another look at this.  What does ` .shift() ` return?  Knowing this what do we need to return from the function?  Do we need to index ` array[0] ` ?